packages/react/test/specs/components/TextArea/TextArea-test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import { mountWithProvider as mount } from 'test/utils'
+import TextArea from 'src/components/TextArea/TextArea'
+import { isConformant } from 'test/specs/commonTests'
+
+const testValue = 'test value'
+
+describe('TextArea', () => {
+  isConformant(TextArea)
+
+  describe('defaultValue', () => {
+    test('sets "defaultValue" as initial "value"', () => {
+      const wrapper = mount(<TextArea defaultValue={testValue} />)
+
+      expect(wrapper.find('textarea').prop('value')).toBe(testValue)
+    })
+  })
+
+  describe('auto-controlled', () => {
```

Comment:
Note sure that it's useful at all now as it tests the same thing as `AutoControlledComponent` does...

Reply:
Does it mean the test code of `Input` is also irrelevant? https://github.com/stardust-ui/react/blob/master/packages/react/test/specs/components/Input/Input-test.tsx#L72

Reply:
Yep, it's strange

Reply:
Ok, I removed it from the other components, #1917